### ISSUE TITLE
Does not work in Cupertino apps

### ISSUE THREAD

**Issue by paldepind** [woltapp/wolt_modal_sheet#61]
Attempting to use `WoltModalSheet` in a Cupertino app results in an error like the following:

```
The following assertion was thrown building WoltBottomSheetDragHandle(dirty, dependencies: [_LocalizationsScope-[GlobalKey#bacba]]):
No MaterialLocalizations found.

WoltBottomSheetDragHandle widgets require MaterialLocalizations to be provided by a Localizations widget ancestor.
The material library uses Localizations to generate messages, labels, and abbreviations.

To introduce a MaterialLocalizations, either use a MaterialApp at the root of your application to include them automatically, or add a Localization widget with a MaterialLocalizations delegate.

The specific widget that could not find a MaterialLocalizations ancestor was: WoltBottomSheetDragHandle
    dirty
    dependencies: [_LocalizationsScope-[GlobalKey#bacba]]
The ancestors of this widget were
    CupertinoTabScaffold
        dependencies: [CupertinoUserInterfaceLevel, MediaQuery, UnmanagedRestorationScope, _InheritedCupertinoTheme]
        state: _CupertinoTabScaffoldState#40ad1
    HomePage
        dependencies: [_CupertinoTabPageStateProvider]
    CupertinoApp
        dependencies: [MediaQuery]
        state: _CupertinoAppState#0015d
```

It seems that the modal requires `MaterialLocalizations` to be provided by a `Localizations` widget whereas a Cupertino app has a `CupertinoLocalizations` instance provided by `Localizations`.

**Comment by paldepind** [comment]
This issue is still present AFAIK.

**Comment by yunusemrebakir** [comment]
Hey @paldepind, can you try Remi's suggestion from [here](https://stackoverflow.com/a/55919047) and let us know if it helps?

**Comment by paldepind** [comment]
Sorry, I'm using a different package now that works with Cupertino.

If you're not interested in supporting Cupertino for this package I would suggest simply stating this in the readme :)

**Comment by ulusoyca** [comment]
@paldepind We released 0.3.0 with instructions on how to use CupertinoApp within the ReadMe file. This is the https://github.com/woltapp/wolt_modal_sheet/pull/120 merged.
